pkg/exchange/batch/batch.go

```diff
@@ -106,8 +112,23 @@ func (e KLineBatchQuery) Query(ctx context.Context, symbol string, interval type
 					return
 				}
 
-				c <- kline
-				startTime = kline.EndTime.Add(time.Millisecond)
+				workingKLines = append(workingKLines, kline)
+
+				if len(workingKLines) == 200 {
+					c <- workingKLines
+					workingKLines = nil
```

Comment:
`workingKLines = workingKLines[:0]`

this preserve the underlying memory but reset the slice.

Reply:
ok